zebra/main.c

```diff
@@ -76,6 +76,9 @@ int keep_kernel_mode = 0;
 u_int32_t nl_rcvbufsize = 4194304;
 #endif /* HAVE_NETLINK */
 
+/* Allow VRF with netns as backend */
+int vrf_with_netns;
+
```

Comment:
Default value should be set here (=0 ?)

Reply:
default value does not pass  style checking scripts ( see https://github.com/FRRouting/frr/pull/1542 for more details)